ci/deps/actions-311-pyarrownightly.yaml

```diff
@@ -0,0 +1,30 @@
+name: pandas-dev
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+
+  # build dependencies
+  - versioneer[toml]
+  - cython>=0.29.33
+
+  # test dependencies
+  - pytest>=7.0.0
+  - pytest-cov
+  - pytest-xdist>=2.2.0
+  - hypothesis>=6.34.2
+  - pytest-asyncio>=0.17.0
+  - boto3
```

Comment:
Is boto3 a required dep for pyarrow?

Reply:
Don't think so. I think i accidentally copied this from another deps file